src/serlio/prtMaterial/PRTMaterialNode.cpp

```diff
@@ -46,6 +47,9 @@ namespace {
 
 constexpr bool DBG = false;
 
+const std::wstring MATERIAL_BASE_NAME = L"serlioGeneratedMaterial";

```

Comment:
it would interfere with material caching in existing scenes, but as we are completely breaking backwards compat with the upcoming release anyways, we can do this.

i've equalized the names.